backend/src/main/java/com/htecgroup/skynest/model/request/BucketEditRequest.java

```diff
@@ -0,0 +1,24 @@
+package com.htecgroup.skynest.model.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class BucketEditRequest {
+
+  @NotBlank(message = "cannot be null or empty")
+  @Size(max = 1000, message = "length cannot be over 1000 characters")
+  private String name;
+
+  @NotBlank(message = "cannot be null or empty")
+  @Size(max = 5000, message = "length cannot be over 5000 characters")
+  private String description;
+
+  private String isPublic;
```

Comment:
no restrictions?

probably fine if this a Boolean/boolean

Reply:
Fixed. There are restrictions now.

Reply:
When this @Pattern is applied it already has to be true or false, Really should be a Boolean